app/webpacker/components/ContactGearOrderPage/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Container, Header } from 'semantic-ui-react';
+import GearOrderForm from './GearOrderForm';
+import i18n from '../../lib/i18n';
+
+export default function ContactGearOrder() {
+  const equipments = [
```

Comment:
"Equipment" doesn't have a plural form.
And this array could be moved out of the component.
```suggestion
  const equipment = [
```